### PR TITLE
Translate new posts into Vietnamese

### PR DESCRIPTION
**Pull request by huynhquancam**
Vietnamese translation for these two posts.

- _posts/2016-03-30-ruby-2-1-9-released.md
- _posts/2016-03-30-ruby-2-1-10-released.md

### CODE REVIEW

File: vi/news/_posts/2016-03-30-ruby-2-1-9-released.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+title: "Phát hành Ruby 2.1.9"
+author: "usa"
+translator: "Cẩm Huỳnh"
+date: 2016-03-30 12:00:00 +0000
+lang: en
+---
+
+Ruby 2.1.9 đã được chính thức phát hành.
+
+Bản phát hành nay bao gồm nhiều bản vá lỗi
```
Comment (joneslee85) [inline vi/news/_posts/2016-03-30-ruby-2-1-9-released.md]: s/nay/này

Comment (joneslee85) [inline vi/news/_posts/2016-03-30-ruby-2-1-9-released.md]: s/bản vá lỗi/vá lỗi

File: vi/news/_posts/2016-03-30-ruby-2-1-9-released.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Xem [ChangeLog](http://svn.ruby-lang.org/repos/ruby/tags/v2_1_9/ChangeLog)
+để biết them chi thêm chi tiết.
```
Comment (joneslee85) [inline vi/news/_posts/2016-03-30-ruby-2-1-9-released.md]: s/them chi thêm chi tiết/thêm chi tiết

File: vi/news/_posts/2016-04-01-ruby-2-1-10-released.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+layout: news_post
+title: "Phát hành Ruby 2.1.10"
+author: "usa"
+translator: "Cẩm Huỳnh"
+date: 2016-04-01 02:00:00 +0000
+lang: en
```
Comment (joneslee85) [inline vi/news/_posts/2016-04-01-ruby-2-1-10-released.md]: s/en/vi

File: vi/news/_posts/2016-03-30-ruby-2-1-9-released.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+---
+layout: news_post
+title: "Phát hành Ruby 2.1.9"
+author: "usa"
+translator: "Cẩm Huỳnh"
+date: 2016-03-30 12:00:00 +0000
+lang: en
```
Comment (joneslee85) [inline vi/news/_posts/2016-03-30-ruby-2-1-9-released.md]: s/en/vi

**Comment by joneslee85** [comment]
thx :)

**Comment by stomar** [comment]
Thanks.

@huynhquancam Note that empty lines in markup start a new paragraph. Fixed with 57b6915e694a2463b5d002f1612c1e6d6f7ea4eb; fixes also links to translated news posts (from `en` to `vi`).

**Comment by huynhquancam** [comment]
@stomar Noted with thanks. 👍

**Pull request merged**